### ISSUE TITLE
about the hyperparameters

### ISSUE THREAD

**Issue by wjxts** [declare-lab/identifiable-transformers#2]
Dear author, thanks for publishing the code. I try to reproduce the exp results in the paper, but  fail to get the accuracy reported in the paper. I wonder if you can tell me the hyperparameters in training transformer. It seems there is only configuration for one dataset ag_news.

**Comment by Bhardwaj-Rishabh** [comment]
Hi! Thank you for your interest in our work! This repository mostly aims to provide a simple identifiable variant of transformers and easy comparison between identifiable and standard transformer variants (for the same set of hyperparameters). In the paper, our aim was not to achieve SOTA but to provide an identifiable variant. There, we might have used a slightly different set of hyperparameters as compared to this repository (which still requires hyperparameter search).

I will be happy to share the exact set of hyperparameters we used in the paper :).
please contact me at rishabhbhardwaj15@gmail.com

**Comment by wjxts** [comment]
Thanks a lot! I have sent you an email. Looking forward to your reply.

**Comment by Bhardwaj-Rishabh** [comment]
Hi!
---
 Rishabh Bhardwaj

**Comment by Bhardwaj-Rishabh** [comment]
Sorry, I found a typo in the email I provided (just corrected). Could you please email me again?

**Comment by wjxts** [comment]
sorry for replying late, I sent you an email again :)
